scripts/soft-reboot

```diff
@@ -0,0 +1,227 @@
+#!/bin/bash
```

Comment:
why do we need this script ?
This doesnt seems to be related to MC-lag CLI, please create a separate PR for this change

Reply:
not required. it came in based on merge. updated main.py to be in sync with master. only change from this PR new mclag.py file and config command addition for mclag in main.py